<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Audittion</class>
 <widget class="QWidget" name="Audittion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="3,2,18">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);
font: 700 36pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>试听列表</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(179, 179, 179,100);
font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
border-radius:15px;</string>
         </property>
         <property name="text">
          <string>播放</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icon/42.png</normaloff>:/icon/42.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(179, 179, 179,100);
font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
border-radius:15px;</string>
         </property>
         <property name="text">
          <string>下载</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icon/23.png</normaloff>:/icon/23.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(179, 179, 179,100);
font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
border-radius:15px;</string>
         </property>
         <property name="text">
          <string>批量</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icon/51.png</normaloff>:/icon/51.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>98</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>搜索</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icon/26.png</normaloff>:/icon/26.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255,0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255,0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255,0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255,0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
